lib/solidus_importer/processors/option_types.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module SolidusImporter
+  module Processors
+    class OptionTypes < Base
+      def call(context)
+        @data = context.fetch(:data)
+        return unless option_type_names.any?
+
+        product = context.fetch(:product)
+        process_option_types(product)
+      end
+
+      private
+
+      def process_option_types(product)
+        option_type_names.each_with_index do |name, i|
+          t = product.option_types.find_or_initialize_by(
+            name: name
+          )
+          t.presentation = name
+          t.name = name.downcase
+          t.position = i + 1
+          t.save
+        end
+      end
+
+      def option_type_names
+        @option_type_names ||= @data.values_at(
+          'Option1 Name',
+          'Option2 Name',
+          'Option3 Name'
+        ).compact.reject { |t| t == 'Title' }
```

Comment:
So we could have a title in options?
It would be nice a comment to explain or an explicit method to filter out that case (ex. `reject_on_title`)